WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java

```diff
@@ -436,10 +423,7 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
             return false;
         }
 
-        if (preference == mLanguagePreference) {
```

Comment:
Question (❓): Just for my FYI, to understand this better, this logic got removed because of the `onLocaleSelected(...)` related change, and the fact that `PerAppLocaleManager.onLanguageChanged(...)` is now handling that too?

PS: I understand that this is transitioning code, but having this `PerAppLocaleManager.onLanguageChanged(...)` within `PerAppLocaleManager` itself, kind of confused be a bit, since I didn't expect this per-app manager class to have any logic related to our own custom in-app language picker feature.

Reply:
Good question! I chose to add `PerAppLocaleManager.onLanguageChanged` to have a central place to handle when the language has changed. Right now it only deals with the user changing the language via our in-app picker, but at some point I'll need to figure out when the language has been changed via system app settings and call this same function.